src/attesters/tdx-ecdsa/collect_evidence.c

```diff
@@ -35,6 +42,47 @@ static int tdx_get_report(const tdx_report_data_t *report_data, tdx_report_t *td
 	return 0;
 }
 
+enclave_attester_err_t tdx_get_tdel_info(enclave_attester_ctx_t *ctx, attestation_evidence_t *evidence)
+{
+    int fd = 0;
+    char tdel_info[TDEL_INFO_SZ];
+
+    RTLS_DEBUG("ctx %p, evidence %p\n", ctx, evidence);
+    if (fd = open(TDEL_INFO, O_RDONLY) < 0) {
+        RTLS_ERR("failed to open TDEL info device\n");
+        return -ENCLAVE_ATTESTER_ERR_INVALID;
+    }
+
+    int tdel_info_sz = read(fd, tdel_info, TDEL_INFO_SZ);
+    if (tdel_info_sz != TDEL_INFO_SZ) {
```

Comment:
How do you know the size of event log is exactly 0x10000 bytes?

Reply:
So you miss to parse tdel_info which can tell the exact size of event log to read. You can consult Liang Zhou for the details.